homeassistant/components/dlna_dmr/media_player.py

```diff
@@ -531,28 +551,13 @@ def usn(self) -> str:
     @property
     def state(self) -> MediaPlayerState | None:
         """State of the player."""
-        if not self._device or not self.available:
+        if not self._device or not self._device.profile_device.available:
```

Comment:
This could be simplified to:
```py
if not self.available:
```

Reply:
I think we can drop the available check here since state is never called if the device is unavailable

Reply:
9d5394693d789f8adc0306fdc113d516d49cdc43